app/models/teacher_resource.rb

```diff
@@ -0,0 +1,31 @@
+class TeacherResource < Resource
+  ALLOWED_CONTENT_TYPES = %w(
+    application/pdf
```

Comment:
Perhaps at a later point we could remove the duplication and move this to a config file?